rule_in_out_errors: str = 'errors'